presidio-analyzer/analyzer/analyzer_engine.py

```diff
@@ -87,14 +100,24 @@ def __remove_duplicates(results):
 
         return filtered_results
 
+    @staticmethod
+    def __remove_low_scores(results, score_threshold):
+        new_results = []
+        for result in results:
+            if result.score >= score_threshold:
+                new_results.append(result)
+
+        return new_results
+
     @classmethod
     def get_language_from_request(cls, request):
         language = request.analyzeTemplate.language
         if language is None or language == "":
             language = DEFAULT_LANGUAGE
         return language
 
-    def analyze(self, correlation_id, text, entities, language, all_fields):
+    def analyze(self, correlation_id, text, entities, language, all_fields,
+                score_threshold=0.1):
```

Comment:
It's nicer to have it as a const, not hard coded
```suggestion
                score_threshold=MIN_THRESHOLD):
```

Reply:
I'm not sure that's the right thing. When working interactively with the code (i.e. in a jupyter notebook or similar), you will not have visibility to the const value. It is customary to have default numeric values for functions. In addition, when importing the function alone (without the module), the code would break for not importing the const.

Reply:
I prefer the const option. how about having the const in this file, so if using this module alone, you still have it?

Reply:
Having the const in the file won't work if you import the function alone. Do you have any reference of places where default function arguments are const?

Reply:
Fixed